remoulade/composition.py

```diff
@@ -257,9 +266,9 @@ def get_results(self, *, block=False, timeout=None):
                 timeout = max(0, int((deadline - time.monotonic()) * 1000))
 
             if isinstance(child, group):
-                yield list(child.get_results(block=block, timeout=timeout))
+                yield list(child.get_results(block=block, timeout=timeout, raise_on_error=raise_on_error))
             else:
-                yield child.get_result(block=block, timeout=timeout)
+                yield child.get_result(block=block, timeout=timeout, raise_on_error=raise_on_error)
 
     def wait(self, *, timeout=None):
```

Comment:
Should accept the `raise_on_error` argument as well, and pass it to `get_results`.